ONSdigital/eq-author
# Tech Debt - Upgrade webpack

### What is the context of this PR?
Upgrade webpack to version 4.

This was done by copying the webpack config from a new app using [CRA 2](https://github.com/facebook/create-react-app)

### How to review 
1. Ensure you can build and run the app locally.
1. Ensure that the app behaves as expected.
1. Ensure that you can still run storybooks.
